dMasterServer/MasterServer.cpp

```diff
@@ -154,15 +155,19 @@ int main(int argc, char** argv) {
 
 		Game::logger->Log("WorldServer", "Found cdclient.fdb.  Converting to SQLite");
 
-		if (FdbToSqlite::Convert(Game::assetManager->GetResPath().string()).ConvertDatabase() == false) {
+		if (FdbToSqlite::Convert(Game::assetManager->GetResPath().string(), (BinaryPathFinder::GetBinaryDir() / "resServer").string()).ConvertDatabase() == false) {
 			Game::logger->Log("MasterServer", "Failed to convert fdb to sqlite");
 			return EXIT_FAILURE;
 		}
+	} else if (!std::filesystem::exists(BinaryPathFinder::GetBinaryDir() / "resServer" / "CDServer.sqlite")) {
+		// If the file doesn't exist in the new CDServer location, move it there
+		Game::logger->Log("MasterServer", "CDServer.sqlite is not located at resServer, but is located at res path.  Moving file...");
+		std::filesystem::copy_file(Game::assetManager->GetResPath() / "CDServer.sqlite", BinaryPathFinder::GetBinaryDir() / "resServer" / "CDServer.sqlite");
```

Comment:
Add a comment that this copies instead of move because the folder may be read only

Reply:
Thats fair - ill add those